packages/sync/composer.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "automattic/jetpack-sync",
+	"description": "A library that allows syncing to the WP.com infrastructure.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"autoload": {
+		"psr-4": {
+			"Automattic\\Jetpack\\Sync\\": "src/"
```

Comment:
It doesn't seem like there is a `src` folder in this package. How does this work?

Reply:
Yeah, I had that locally, but didn't end up committing it 😉 It's in the branch now.